common/components/chrome/AlertHeader.jsx

```diff
@@ -178,10 +182,8 @@ class AlertHeader extends React.Component<Props, State> {
 
   _renderTriggeredAlert(): React$Node {
     const key: string = _.keys(AlertMessages).find(key => url.argument(key));
-    const message: string = AlertMessages[key].replace(
-      "{value}",
-      decodeURI(url.argument(key))
-    );
+    const message: string = AlertMessages[key] + '"' + url.decodeNameFromUrlPassing(url.argument(key)) + '"' 
+                          + AlertMessageEnding;
```

Comment:
Avoiding duplicating code is generally good, but in this case I think the added code complexity to reduce duplication, coupled with difficulty future code editors might have in searching for these now chopped up front-end strings, would make me prefer the old approach.

Reply:
Agreed, and I did this to deal with Javascript's strange replace() functionality, which interprets sequences such as $& as flags.

In the end I realized that I can very easily void this by just performing the string replacement when the project/event/group name is still URL encoded.